<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCloudForm</class>
 <widget class="QWidget" name="QCloudForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget_cloud">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widgetCloud" native="true">
         <widget class="QCPushButton" name="pushButton_del">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>320</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
         <widget class="QCPushButton" name="btn_print">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>320</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
         <widget class="QCPushButton" name="pushButton_down">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>320</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
         <widget class="QCPushButton" name="pushButton_upload">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>320</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>4</y>
            <width>96</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Cloud Service:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>0</y>
            <width>92</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Local Service:</string>
          </property>
         </widget>
         <widget class="QCloudTreeWidget" name="treeCloud">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>40</y>
            <width>321</width>
            <height>251</height>
           </rect>
          </property>
         </widget>
         <widget class="QCloudLocalTreeWidget" name="treeLocal">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>40</y>
            <width>321</width>
            <height>251</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCPushButton</class>
   <extends>QPushButton</extends>
   <header>qcpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCloudTreeWidget</class>
   <extends>QTreeView</extends>
   <header>qcloudtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCloudLocalTreeWidget</class>
   <extends>QTreeView</extends>
   <header>qcloudlocaltreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>heartbeatclick()</slot>
  <slot>getdirclick()</slot>
  <slot>getfilesclick()</slot>
  <slot>upload()</slot>
 </slots>
</ui>
